crates/invoker-impl/src/error.rs

```diff
@@ -99,6 +99,9 @@ pub(crate) enum InvokerError {
     #[error("error when trying to read the journal: {0}")]
     #[code(restate_errors::RT0006)]
     JournalReader(anyhow::Error),
+    #[error("error when trying to read the journal: no journal present")]
+    #[code(unknown)]
+    NoJournalPresent,
```

Comment:
Should this maybe be `NotInvoked`? Theoretically the journal might still be present (e.g. when retaining the journal after completion/deletion/killing or when being in state suspended).